configs/machines/levante.yaml

```
@@ -72,7 +72,8 @@ computer:
     pool_dir:  "/pool/data"
 
     # kh 15.02.22 preferred on levante (at the moment), on mistral intelmpi was the default
-    compiler_mpi: intel2022_openmpi
+    #compiler_mpi: intel2022_openmpi
+    compiler_mpi: gnu11_ompi4 
```

```suggestion
    compiler_mpi: intel2022_openmpi 
```

This might break AWI models because they might all depend on this default. Either, make sure Geomar models setup this in their relevant files and revert this change or check that AWI models all specify their `compiler_mpi` parameter in their files. Check esm_tests changes